monorepo-migration/sync-ac.sh

```diff
@@ -67,7 +77,8 @@ function _remove_old_tags() {
 
 function _merge_histories() {
     cd "$SCRIPT_DIR"
-    git pull --tags --allow-unrelated-histories --no-rebase --force "$REPO_PATH"
+    git pull --no-edit --tags --allow-unrelated-histories --no-rebase --force "$REPO_PATH"
+    git commit --amend --message "$MERGE_COMMIT_MESSAGE"
```

Comment:
`git pull` doesn't take `--message` as an argument. `--no-edit` just fills the regular commit message. The next line customize the message.